biproductive/templates/static/scripts/game.js

```diff
@@ -0,0 +1,298 @@
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981117/memory/starwars/dark%20sidious.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981141/memory/starwars/dark%20vador.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981165/memory/starwars/padme.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981190/memory/starwars/leia.jpg'
+    ],
+    lotr: [
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981408/memory/lotr/gandalf.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981438/memory/lotr/sauron.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981469/memory/lotr/Aragorn.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981501/memory/lotr/legolas.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981535/memory/lotr/Gimli.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981603/memory/lotr/golum.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981645/memory/lotr/sam.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981686/memory/lotr/saroumane.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981738/memory/lotr/bilbo.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981802/memory/lotr/frodo.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981408/memory/lotr/gandalf.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981438/memory/lotr/sauron.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981469/memory/lotr/Aragorn.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981501/memory/lotr/legolas.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981535/memory/lotr/Gimli.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981603/memory/lotr/golum.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981645/memory/lotr/sam.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981686/memory/lotr/saroumane.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981738/memory/lotr/bilbo.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981802/memory/lotr/frodo.jpg'
+    ],
+    disney: [
+        'https://res.cloudinary.com/beumsk/image/upload/v1547982044/memory/disney/mickey.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547982088/memory/disney/mowgli.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547982610/memory/disney/tarzan.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547982620/memory/disney/simba.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547982628/memory/disney/aladin.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547982636/memory/disney/blanche%20neige.jpg',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rizvansky/biproductive/pullRequest?prid=8150207)
```suggestion
        "https://res.cloudinary.com/beumsk/image/upload/v1547982636/memory/disney/blanche%20neige.jpg",
```